nzediegwu1/EventManager
# Setup sequelize

#### What does this PR do?
Setup sequelize

#### Description of Task to be completed?
Install sequelize and sequelize-cli
Create and configure sequelizerc
Initialize sequelize in server directory
Refractor sequelize config and model index files  respectively to ES6
Enter EventManager db credentials into the sequelize config file created
Create models for users, events, and centers using sequelize-cli commands
Refractor model and migration files created using ES6 and updated changes in documentation
Define model relationships within the model and migration files created
Implement migration files using sequelize db:migrate command

#### How should this be manually tested?
Run migration, using: sequelize db:migrate, then check the database table schemas to confirm that they reflect the models created and relationships defined

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories? (If applicable)
https://www.pivotaltracker.com/story/show/152858511

#### Screenshots (If applicable)
None
